addons/stock_account/views/stock_account_views.xml

```diff
@@ -51,5 +51,19 @@
             </field>
         </record>
 
+        <record id="view_picking_form_inherit_stock_account" model="ir.ui.view">
```

Comment:
I don't really understand the point of this? Can you add a code comment explaining why this is needed? Especially the invisible field since invisible fields should be auto-added unless needed for a special purpose (and if they are needed for a special purpose then there should be a code comment after it as per this commit's description: https://github.com/odoo/odoo/commit/5639ed865c5a3b82bab25b0ecac28c68c02e9306 )

Reply:
removed the invisible field (forgot this pr, thank you)